CoroutineTemplate/app/build.gradle

```diff
@@ -87,6 +93,9 @@ dependencies {
 
     testImplementation(
         "junit:junit:$test_junit_version",
+        "org.amshove.kluent:kluent-android:$test_kluent_version",
+        "com.nhaarman.mockitokotlin2:mockito-kotlin:$test_mockito_version",
+        "org.robolectric:robolectric:$test_robolectric_version"
```

Comment:
Let's keep everything related to testing out of this PR for now 🙏 We'll work on that in a separate ticket

The reason is because we might want to use different libraries (Ex: `mockk` instead of `mockito`).

What do you guys think? 🙏

Reply:
IMO, I think we could keep the unit testing as is for now just to note that we should have these test cases to cover the main functionality of this PR. Then we can refactor it with a different library later.

What do you think @Tuubz 🙏🏼

Reply:
I just like the idea of having everything related to testing in a separate ticket (As this ticket is meant for navigation)

Ex: Let's say in the future, we want to trace back in which PR the testing environment was set-up, we wouldn't be expecting to see it in a PR for navigation component. 

But no biggie. We can keep it and re-factor it later, works for me 🙏

Reply:
I see, thanks @Tuubz, removed the unit testings accordingly 🙏🏼